wsagent/agent/src/main/resources/org.eclipse.che.ws-agent.script.sh

```diff
@@ -55,7 +55,7 @@ MACHINE_TYPE=$(uname -m)
 
 mkdir -p ${CHE_DIR}
 ${SUDO} mkdir -p /projects
-${SUDO} sh -c "chown -R $(id -u -n) /projects"
+${SUDO} sh -c "find /projects -not -uid $(id -u -n) -print0 | xargs -0 -P3 --no-run-if-empty chown $(id -u -n )"
```

Comment:
```
user@ce7b8a5773c0:/projects$ find /projects -not -uid $(id -u -n) -print0
find: invalid argument `user' to `-uid'
user@ce7b8a5773c0:/projects$ echo $?
1
```

Reply:
@monaka 
I guess something is missed

Reply:
it should be either `-uid $(id -u)` or `-user $(id -u -n)` First option is better I think.

@monaka can you push changes please and will merge this one?

Reply:
I see. I'll push again.